tests/check_consistent.py

```diff
@@ -46,6 +46,7 @@ def check_stdlib() -> None:
 
 def check_stubs() -> None:
     for distribution in os.listdir("stubs"):
+        assert not distribution.startswith("types-"), f"Distribution not allowed to start with 'types-': {distribution}"
         assert not os.path.isfile(distribution), f"Only directories allowed in stubs, got {distribution}"
```

Comment:
Hmm, this check looks buggy. It should be checking `os.path.isfile(os.path.join("stubs", distribution))`. I'll fix in another PR.